c1/src/lib.rs

```diff
@@ -49,16 +55,16 @@ const SINGLE_CALL_GAS: u64 = 100_000_000_000_000;
 #[near_bindgen]
 impl Con1 {
   /// Deploy Con2 from within Con1 to a new address: c3.YOURADDRESS.testnet.
-  pub fn deploy_con2_to(&self, subaddress: String) {
```

Comment:
why remove?

Reply:
I was inspecting code with `carge-expand` and wanted to make it minimal.